frontend/containers/account-pending-approval/component.tsx

```diff
@@ -33,7 +35,7 @@ export const AccountPendingApproval = () => {
       <Button className="mt-6 mb-12" to="/discover">
         <FormattedMessage defaultMessage="Explore" id="7JlauX" />
       </Button>
-      <Link href="/faq#pending-approval" passHref>
+      <Link href={FaqPaths['why-is-my-account-pending-approval']} passHref>
```

Comment:
I think we should link the paths like so: 

```
FaqPaths[FaqQuestions.WhyAccountIsPendingApproval]
```

instead. It'd allow us to change the urls in the future without breaking the links.

Reply:
Ok! I will change them